packages/ui/primitives/document-flow/document-flow-root.tsx

```diff
@@ -61,8 +61,14 @@ export const DocumentFlowFormContainerContent = ({
   ...props
 }: DocumentFlowFormContainerContentProps) => {
   return (
-    <div className={cn('flex flex-1 flex-col', className)} {...props}>
-      <div className="-mx-2 flex flex-1 flex-col overflow-y-auto px-2">{children}</div>
+    <div
+      className={cn(
+        `flex flex-1 flex-col overflow-y-auto overflow-x-hidden [&::-webkit-scrollbar]:hidden`,
+        className,
+      )}
+      {...props}
+    >
+      <div className="-mx-2 flex flex-1 flex-col px-2">{children}</div>
     </div>
```

Comment:
> I'm facing one small issue tho, kinda stuck on it. Could you please lmk what I can do here so that the input field gets borders on all sides or perhaps how to prevent the scollbar from pushing the elements to the left?

You can try this:

```
    <div className={cn('-mx-2 flex flex-1 flex-col overflow-y-auto px-2', className)} {...props}>
      {children}
    </div>
```

Reply:
Oki Doki, will try that one :)